plugins/reach/lib/kReachManager.php

```diff
@@ -31,7 +31,9 @@ protected function getObjectType($eventObjectClassName)
 			"permissionItem" => objectType::PERMISSIONITEM,
 			"userRole" => objectType::USERROLE,
 			"categoryEntry" => objectType::CATEGORY_ENTRY,
-			"CaptionAsset" => CaptionAssetEventNotificationsPlugin::getEventNotificationEventObjectTypeCoreValue('CaptionAsset'),);
+			"CaptionAsset" => CaptionAssetEventNotificationsPlugin::getEventNotificationEventObjectTypeCoreValue('CaptionAsset'),
```

Comment:
Use predefined const CaptionAssetEventNotificationEventObjectType::CAPTION_ASSET
same for transcription